schemas/test/en/test_list_collector_repeating_blocks.json

```diff
@@ -0,0 +1,227 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test a Simple List Collector with Repeating Blocks",
+    "theme": "default",
+    "description": "A questionnaire to test a simple list collector with repeating blocks and no additional blocks or sections.",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "answer_codes": [
```

Comment:
I'm not sure the answer_codes are behaving as expected. When I test functionally even though I have answered the questions for repeating blocks they don't appear to be in the submission payload.

Looks like this will need a fix and some tests

Reply:
Great catch thanks Rhys! I had only been checking the debug dump, didn't realise those values weren't being added to the submission